app/views/play/level/modal/CourseVictoryComponent.vue

```diff
@@ -180,6 +185,11 @@
             },
             []
         )
+        if me.showHeroAndInventoryModals()
+          @setupManager?.destroy()
+          @setupManager = new LevelSetupManager supermodel: @supermodel, levelID: @nextAssessment.slug, levelPath: "level", hadEverChosenHero: true, parent: @parent, courseID: @course._id, courseInstanceID: @courseInstanceID,invokedFromCourseVictory:true
+          unless @setupManager?.navigatingToPlay
+            @setupManager.open()
```

Comment:
Will this mistakenly open the setup manager if loading the level takes a long time?

Looking at LevelSetupManager, it seems like navigatingToPlay won't be set until the level is loaded, so there's a period of time to mistakenly open the setup manager here.  Unless you know for sure the level will always be loaded at this point.

Reply:
Looking at this again, I don't think the level loading matters.  `navigatingToPlay` will only be set to true if `me.showHeroAndInventoryModalsToStudents()` is false, and we only go down this code path if `me.showHeroAndInventoryModalsToStudents()` is true.